index.html

```diff
@@ -288,10 +299,15 @@ <h2>
             </li>
             <li>Set [=request/Destination=] to "image".
             </li>
-            <li>Set [=request/Mode=] to "no-cors".
+            <li>Set [=request/Client=] to the [=context object=]'s [=relevant settings object=].
             </li>
           </ol>
         </li>
+        <li>If |request|'s [=request/Client=] is null, throw a {{TypeError}}.
+        </li>
+        <aside class="note">
```

Comment:
I don’t think an `aside` is valid within a list element.

Reply:
Fair, I moved it below the steps